Added README for Rack Instrumentation (#389)
open-telemetry/opentelemetry-ruby: instrumentation/rack/README.md

# OpenTelemetry Rack Instrumentation

> The Rack instrumentation is a community-maintained instrumentation for the [Rack][rack-home] Ruby jobs system.

Reviewer: ```suggestion
The Rack instrumentation is a community-maintained instrumentation for the [Rack][rack-home] Ruby web server interface.
```

## How do I get started?

Install the gem using:

```
gem install opentelemetry-instrumentation-rack
```

Or, if you use [bundler][bundler-home], include `opentelemetry-instrumentation-rack` in your `Gemfile`.

## Usage

To use the instrumentation, call `use` with the name of the instrumentation:

```ruby
OpenTelemetry::SDK.configure do |c|

>   c.use 'OpenTelemetry::Instrumentation::rack'

Reviewer: ```suggestion
  c.use 'OpenTelemetry::Instrumentation::Rack'
```

end
```

Alternatively, you can also call `use_all` to install all the available instrumentation.

```ruby
OpenTelemetry::SDK.configure do |c|
  c.use_all
end
```

## How can I get invovled?

The `opentelemetry-instrumentation-rack` gem source is [on github][repo-github], along with related gems including `opentelemetry-api` and `opentelemetry-sdk`.

The OpenTelemetry Ruby gems are maintained by the OpenTelemetry-Ruby special interest group (SIG). You can get involved by joining us on our [gitter channel][ruby-gitter] or attending our weekly meeting. See the [meeting calendar][community-meetings] for dates and times. For more information on this and other language SIGs, see the OpenTelemetry [community page][ruby-sig].

## License

The `opentelemetry-instrumentation-rack` gem is distributed under the Apache 2.0 license. See [LICENSE][license-github] for more information.

> [rack-home]: https://github.com/mperham/rack

Reviewer: ```suggestion
[rack-home]: https://github.com/rack/rack
```
